src/gui/fspathedit_p.cpp

```diff
@@ -251,7 +251,7 @@ void Private::FileLineEdit::keyPressEvent(QKeyEvent *e)
         showCompletionPopup();
     }
 
-    auto *validator = qobject_cast<const FileSystemPathValidator *>(this->validator());
+    const auto *validator = qobject_cast<const FileSystemPathValidator *>(this->validator());
```

Comment:
We discussed this line in previous PR. `auto` will give you the same result.

Reply:
Yeah sorry, didn't even look what's on the right hand here.

Reply:
Reverted.